### ISSUE TITLE
Consumes a lot of GPU resources

### ISSUE THREAD

**Issue by salos1982** [microsoft/vscode#170609]
Type: <b>Bug</b>

I'm working in node.js project and have opened 3 Visual Studio Windows with 3 projects (2 bacend and on frontend). And I hear my laptop is very noisy. I opened task manager and found that Visual Studio Code uses about 17% of my GPU Nvidia RTX 3060. There is no any animation on screen? no any graphics at all. Why Visual Studio Code consumes so much GPU? 

VS Code version: Code 1.74.2 (e8a3071ea4344d9d48ef8a4df2c097372b0c5161, 2022-12-20T10:29:14.590Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5800HS with Radeon Graphics          (16 x 3194)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|39.41GB (20.79GB free)|
|Process Argv|--crash-reporter-id 20b654f9-6fe3-46e0-8b3e-8bc4311e9792|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (51)</summary>

Extension|Author (truncated)|Version
---|---|---
tsl-problem-matcher|amo|0.6.2
npm-intellisense|chr|1.4.4
vscode-eslint|dba|2.2.6
vscode-springboot-developer-pack|dev|1.0.3
vscode-springboot-snippets|dev|1.2.0
gitlens|eam|13.2.0
vscode-npm-script|eg2|0.3.29
vscode-jest-runner|fir|0.4.59
vscode-pull-request-github|Git|0.56.0
gc-excelviewer|Gra|4.2.56
vscode-graphql|Gra|0.8.4
vscode-graphql-syntax|Gra|1.0.5
vscode-test-explorer|hbe|2.21.1
search-node-modules|jas|1.3.0
vscode-sbt-scala|lig|0.2.4
vscode-language-babel|mgm|0.0.36
vscode-docker|ms-|1.23.3
vscode-kubernetes-tools|ms-|1.3.11
isort|ms-|2022.8.0
python|ms-|2022.20.1
vscode-pylance|ms-|2023.1.10
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.266.1
remote-wsl|ms-|0.72.0
test-adapter-converter|ms-|0.1.6
vscode-typescript-next|ms-|5.0.202301040
back-n-forth|nic|3.1.1
vscode-boot-dev-pack|Piv|0.2.0
vscode-spring-boot|Piv|1.43.0
java|red|1.13.0
vscode-commons|red|0.0.6
vscode-yaml|red|1.10.1
modulegenerator|Ser|1.1.2
java-generate-setters-getters|soh|7.4.0
intellicode-api-usage-examples|Vis|0.2.6
vscodeintellicode|Vis|1.2.29
vscode-java-debug|vsc|0.47.0
vscode-java-dependency|vsc|0.21.1
vscode-java-pack|vsc|0.25.7
vscode-java-test|vsc|0.37.1
vscode-maven|vsc|0.40.2
vscode-spring-boot-dashboard|vsc|0.9.0
vscode-spring-initializr|vsc|0.11.1
gitblame|wad|10.1.0
nodejs-extension-pack|wad|0.1.9
vscode-wakatime|Wak|22.1.0
JavaScriptSnippets|xab|1.8.0
markdown-all-in-one|yzh|3.5.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1:30587005
3biah626:30602489
3d0df643:30613357
f6dab269:30613381
fim-prod:30623723

```

</details>

**Comment by salos1982** [comment]
One important comment. I found that the issue appears only after starting a terminal (I use hit-bash as a terminal by default)

**Comment by deepak1556** [comment]
@salos1982 does the issue repro when you run with the following setting `"terminal.integrated.gpuAcceleration": "off"` ?

**Comment by salos1982** [comment]
Yes, it reproduces even when I set "terminal.integrated.gpuAcceleration": "off"

**Comment by salos1982** [comment]
[test1.zip](https://github.com/microsoft/vscode/files/10365729/test1.zip)
I've created a sample project. All you need is to install dependencies using `npm ci` and then  run `npm run dev`

**Comment by salos1982** [comment]
VSCode 1.74.3 does not consume so much GPU.

**Comment by salos1982** [comment]
I do not know why but after restart of my laptop it again consumes a lot of GPU

**Comment by deepak1556** [comment]
Can you check if the issue persists with our insiders version https://code.visualstudio.com/insiders

**Comment by salos1982** [comment]
Yes, now it's much better. Thanks

**Comment by deepak1556** [comment]
Thanks for confirming, the changes will be available with upcoming `1.75` release. Closing this issue for now, can reopen if the issue persists.

**Comment by salos1982** [comment]
It is strange but the update to version 1.75 didn't help. But a fresh install helped.
